<launch>
  <arg name="map_file" default="$(find utexas_gdc)/maps/3ne-real-new.yaml" />
  <arg name="door_file" default="$(find segbot_logical_translator)/config/real_doors.yaml" />
  <arg name="location_file" default="$(find segbot_logical_translator)/config/real_locations.yaml" />
  <node name="segbot_logical_translator_gui" type="segbot_logical_translator_gui" pkg="segbot_logical_translator_gui">
    <param name="map_file" value="$(arg map_file)" />
    <param name="door_file" value="$(arg door_file)" />
    <param name="location_file" value="$(arg location_file)" />
    <param name="values_file" value="$(find segbot_logical_translator_gui)/data/values" />
    <param name="lua_file" value="$(find segbot_logical_translator_gui)/scripts/distances.lua" />
    <param name="sim_auto_door" value="false" />
  </node>
</launch>
